### PR TITLE
Add TermFinance Avalanche network deployment

### PR DESCRIPTION
**Pull request by robotoer**
This PR adds support for tracking TermFinance terms on Avalanche.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/term-finance exports TVL: 
        
 
 
    ethereum-borrowed         75.09 M
    borrowed                  75.09 M
    ethereum                  19.63 M
    
    total                    19.63 M

File: projects/term-finance/index.js [hunk 1]
```diff
@@ -43,6 +43,9 @@ const emitters = {
     "0xf268E547BC77719734e83d0649ffbC25a8Ff4DB3",  // 0.4.1
     "0xc60e0f5cD9EE7ACd22dB42F7f56A67611ab6429F",  // 0.6.0
   ],
+  "avalanche": [
+    "0xb81afB6724ba9d19a3572Fb29ed7ef633fD50093",  // 0.6.0
```
Comment (g1nt0ki) [inline projects/term-finance/index.js:47]: can you update code so start block also changes based on chain?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
thanks the mostly looks good except for the start block

File: projects/term-finance/index.js (hunk 1 above)
Comment (robotoer) [reply projects/term-finance/index.js:50]: Makes sense! Should be updated now

**Comment by llamatester** [comment]
The adapter at projects/term-finance exports TVL: 
        
 
 
    ethereum-borrowed         74.90 M
    borrowed                  74.90 M
    ethereum                  19.62 M
    
    total                    19.62 M

**Comment by g1nt0ki** [comment]
Thanks again for the PR

**Pull request merged**